test/HealthChecks.Nats.Tests/Functional/DefaultStartup.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Microsoft.AspNetCore.Builder;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace HealthChecks.Nats.Tests.Functional
+{
+    public class DefaultStartup
```

Comment:
You can delete this file. I did it for all such files in one of the latest commits - #955

Reply:
Commit: 20198911678fd344d68f0bcae7e6da5fbb88a7ff [2019891]
Parents: d4ae8d388b
remove default startup